### PR TITLE
refactor: Adapt apis to integrate Spark in the Adevinta Platform

### PR DESCRIPTION
**Pull request by soulcramer**
## 📋 Changes description

Adapt some api to integrate Spark in the Adevinta Platform

## 🤔 Context

We now have the LBC color palette and can integrate Spark into the Adevinta Platform but we need to make some adjustments to make it compatible with the compose migration and ensure that we don't break the UI with the new tokens

## 🗒️ Other info

Related to #215

### CODE REVIEW

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 1]
```diff
@@ -138,10 +127,10 @@ public fun lightSparkColors(
...
     secondaryVariant: Color = SugarCotton30,
     onSecondaryVariant: Color = White,
-    tertiary: Color = BrikkeGrey,
-    onTertiary: Color = GreyBlue99,
-    tertiaryContainer: Color = BrikkeGreyExtraLight,
-    onTertiaryContainer: Color = GreyBlue10,
+//    tertiary: Color = BrikkeGrey,
+//    onTertiary: Color = GreyBlue99,
+//    tertiaryContainer: Color = BrikkeGreyExtraLight,
+//    onTertiaryContainer: Color = GreyBlue10,
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:133]: Should we remove them completely?

```suggestion
```

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 2]
```diff
@@ -248,10 +249,10 @@ public fun darkSparkColors(
...
     secondaryVariant: Color = SugarCotton70,
     onSecondaryVariant: Color = Black,
-    tertiary: Color = GreyBlue40,
-    onTertiary: Color = GreyBlue90,
-    tertiaryContainer: Color = GreyBlue20,
-    onTertiaryContainer: Color = GreyBlue95,
+//    tertiary: Color = GreyBlue40,
+//    onTertiary: Color = GreyBlue90,
+//    tertiaryContainer: Color = GreyBlue20,
+//    onTertiaryContainer: Color = GreyBlue95,
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:255]: Same
```

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 3]
```diff
@@ -833,7 +899,12 @@ public class SparkColors(
         append("onNeutralContainer=$onNeutralContainer, ")
         append("inversePrimary=$inversePrimary, ")
         append("inverseSurface=$inverseSurface, ")
-        append("inverseOnSurface=$inverseOnSurface, ")
+        append("dim1=$dim1, ")
+        append("dim2=$dim2, ")
+        append("dim3=$dim3, ")
+        append("dim4=$dim4, ")
+        append("dim5=$dim5, ")
+        append(", ")
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:907]: To be removed

File: spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt [hunk 4]
```diff
@@ -179,6 +179,7 @@ internal object PaletteTokens {
     internal val Surfer80 = Color(0xFFD1D0D5)
     internal val Surfer90 = Color(0xFFEBEBED)
     internal val Surfer95 = Color(0xFFF8F8F9)
+    internal val Surfer975 = Color(0xFFF8F8F9)
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt:182]: This seems to be a duplicate of `Surfer95 `

**Review by SimonMarquis — CHANGES_REQUESTED** [review]

File: spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt (hunk 4 above)
Comment (soulcramer) [reply spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt:181]: Yep it's `Surfer95` that has the wrong name and should be deleted instead

**Review by SimonMarquis — APPROVED** [review]

**Pull request merged**